installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc

```diff
@@ -0,0 +1,26 @@
+[id="installing-azure-stack-hub-account"]
+= Configuring an Azure Stack Hub account
+include::modules/common-attributes.adoc[]
+:context: installing-azure-stack-hub-account
+
+toc::[]
+
+Before you can install {product-title}, you must configure a Microsoft Azure account.
+
+[IMPORTANT]
+====
+All Azure resources that are available through public endpoints are subject to resource name restrictions, and you cannot create resources that use certain terms. For a list of terms that Azure restricts, see link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-manager-reserved-resource-name[Resolve reserved resource name errors] in the Azure documentation.
+====
+
+include::modules/installation-azure-limits.adoc[leveloffset=+1]
```

Comment:
This does not look relevant to Azure Stack. I believe this would be the equivalent for ASH: https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-quota-types?view=azs-2102

Reply:
The part I am talking about is the part that says: "
<div class="paragraph" style="box-sizing: border-box; text-rendering: optimizelegibility; -webkit-font-smoothing: antialiased; color: rgb(0, 0, 0); font-family: RedHatText, Overpass, &quot;Open Sans&quot;, Helvetica, Arial, sans-serif; font-size: 18px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><p style="box-sizing: border-box; text-rendering: optimizelegibility; -webkit-font-smoothing: antialiased; margin: 0px 0px 0.92em; color: rgb(84, 84, 84); line-height: 1.6; font-size: 0.92em;">The OpenShift Container Platform cluster uses a number of Microsoft Azure components, and the default<span> </span><a href="https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits" style="box-sizing: border-box; text-rendering: optimizelegibility; -webkit-font-smoothing: antialiased; background-color: transparent; color: rgb(34, 117, 180); text-decoration: none;">Azure subscription and service limits, quotas, and constraints</a><span> </span>affect your ability to install OpenShift Container Platform clusters.</p></div><div class="admonitionblock important" style="box-sizing: border-box; text-rendering: optimizelegibility; -webkit-font-smoothing: antialiased; background: rgba(238, 33, 0, 0.082); border-left: solid rgb(238, 33, 0); color: rgb(0, 0, 0); font-family: RedHatText, Overpass, &quot;Open Sans&quot;, Helvetica, Arial, sans-serif; font-size: 18px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

  | Default limits vary by offer category types, such as Free Trial and Pay-As-You-Go, and by series, such as Dv2, F, and G. For example, the default for Enterprise Agreement subscriptions is 350 cores.Check the limits for your subscription type and if necessary, increase quota limits for your account before you install a default cluster on Azure.
-- | --


</div>"